reference/conanfile_txt.rst

```diff
@@ -114,18 +114,15 @@ List of files to be imported to a local directory. Read more: :ref:`imports<impo
 
 The first item is the subfolder of the packages (could be the root "." one), the second is the pattern to match. Both relate to the conan local cache. The third (after the arrow) item, is the destination folder, living in user space, not in the conan cache.
 
-
-The ``[imports]`` section also support the same arguments as the equivalent ``imports()`` method in conanfile.py, separated with an @
-
-- root_package: fnmatch pattern of the package name ("OpenCV", "Boost") from which files will be copied. Default: all packages in deps
```

Comment:
I think I'd keep here a copy of the arguments, linking conanfile.txt users to conanfile.py could be confusing to them.

Reply:
I think it is harder to maintain this way as explanations need to match in both places. But it is ok, as it is something that is not willing to change in the future.